docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# vim: set ft=dockerfile:
+FROM ruby:2.6-slim
+
+ENV DEBIAN_FRONTEND="noninteractive" \
```

Comment:
we can set these values in the application.yml file that specifying it.

Reply:
typically environment variables are preferred in docker

Reply:
@ptman so right now the gem `figaro` loads whatever in application.yml and converts it into ENV variables.  we should figure out the best way here.   So that the env variable configurations is not scattered across. 

also, the compose file will be a better place to store the env variables?

Reply:
I know docker way better than I know rails. Where are these magic variables documented? They should be documented for docker image users, whether they use docker-compose or not and are familiar with rails or not.

Reply:
@ptman If you are looking at the documentation for all variables we have please have a look at https://www.chatwoot.com/docs/environment-variables

Please let me know if something specific needs to be addressed in terms of documentation.